traits/etsconfig/tests/test_etsconfig.py

```diff
@@ -8,7 +8,10 @@
 import sys
 import tempfile
 import time
-import unittest
+if sys.version_info[:2] == (2, 6):
+    import unittest2 as unittest
+else:
+    import unittest
```

Comment:
Elsewhere in the library we use `from traits.testing.unittest_tools import unittest`, which does the same but without repetitions